test/karma/launchers.js

```diff
@@ -257,7 +257,8 @@ module.exports = {
 		base: "BrowserStack",
 		device: "iPhone 6S",
 		os: "ios",
-		os_version: "11.4"
+		os_version: "11.4",
+		real_mobile: true
```

Comment:
@sd5869 Isn't the same needed for the iOS 12.0 entry below?

Reply:
Yes it is needed should I add in that also ?

Reply:
Note: This will be needed for all devices where the entry in ```real_devices``` in our API

Reply:
Yes, please add it there as well.

Will it still be needed in a few months when you drop simulators and all devices will be real?

Reply:
I have added it.